startup/30-detectors.py

```diff
@@ -90,7 +90,7 @@ def set_primary(self, n, value=None):
 class ESMbpm(ESMQuadEM):
     conf = Cpt(QuadEMPort, port_name="NSLS2_EM")
 
-
+#uncomment later and fix
```

Comment:
@maffettone, do we need this comment? Looks like everything works OK for QuadEM's.

Reply:
Don't think so. Not sure where it came from (potentially a bad copy/paste job) by the look of it.